config/locales/strings/en.ads.yml

```diff
@@ -2,7 +2,7 @@ en:
   ads:
     advertisements: 'advertisements'
     blurb_index: >
-      You can, if you want, advertise this Codidact community on your personal homepage or on other sites
+      You can, if you want, advertise this Codidact community on your personal homepage or on other communities
       (with the permission of the providers!)
```

Comment:
This one is an incorrect replacement IMHO, as it's not limited to other Codidact communities.

Reply:
Agree with luap.

Reply:
Issue addressed - change reverted.